src/index.js

```diff
@@ -10,44 +17,59 @@ export default function livereload (options = { watch: '' }) {
     options.watch = options.watch || ''
   }
 
-  let enabled = options.verbose === false
+  const verbose = options.verbose !== false
   const port = options.port || 35729
-  const snippetSrc = options.clientUrl ? JSON.stringify(options.clientUrl) : `'//' + (window.location.host || 'localhost').split(':')[0] + ':${port}/livereload.js?snipver=1'`
-  const server = createServer(options)
+  const snippetSrc = options.clientUrl
+    ? JSON.stringify(options.clientUrl)
+    : `'//' + (window.location.host || 'localhost').split(':')[0] + ':${port}/livereload.js?snipver=1'`
 
-  // Start watching
-  if (Array.isArray(options.watch)) {
-    server.watch(options.watch.map(w => resolve(process.cwd(), w)))
-  } else {
-    server.watch(resolve(process.cwd(), options.watch))
+  if (state === 0) {
+    start()
+  } else if (state === 1) {
+    // Starting, just ignore
```

Comment:
I feel a bit uneasy about this. Seems like a hole a race condition could sneak in...

I acknowledge it's very unlikely (that the livereload server takes more time to bind, than Rollup would take to restart)... But I would have preferred impossible.

Reply:
I thought that by tracking the state of the server, it's easier to avoid multiple servers trying to bind at the same time. Like you say, what if the server takes more time to bind? => multiple servers will try to get the same port. This code makes that impossible.

In the ideal case (if state=1) it should wait until the previous server is started, then stop it and then start a new one with the latest config. But that seems like overengineering.

Reply:
You're right, that seems very reasonable.